nautobot/extras/tests/test_api.py

```diff
@@ -445,6 +453,7 @@ def run(self, data, commit=True):
 
             return "Job complete"
 
+        # TODO what is this doing here
```

Comment:
If we had a test that actually inspects the JobResult generated from running this TestJob, this would make sense as a functional test (making sure that `test_*` functions are automatically invoked as expected). Not relevant to API testing though - I'm fine if you want to remove this.